pandas/tests/indexes/datetimes/methods/test_normalize.py

```diff
@@ -70,6 +70,7 @@ def test_normalize_tz(self):
         assert not rng.is_normalized
 
     @td.skip_if_windows
+    @td.skip_if_wasm  # tzset is available only on Unix-like systems
```

Comment:
Instead of `@td.skip_if_wasm`, could you use `@pytest.mark.skipif(WASM, reason=your comment here")` instead? (Make it clearer in the logs)

Reply:
Addressed in 9f465280a0b91ca7a180b1b8e894d5a4b7b77c1f.